src/ethereum_test_forks/tests/test_forks.py

```diff
@@ -153,6 +161,50 @@ def test_forks():
     assert fork == Berlin
 
 
+def test_transition_fork_comparison():
+    """
+    Test comparing to a transition fork.
+
+    The comparison logic is based on the logic we use to generate the tests.
+
+    E.g. given transition fork A->B, when filling, and given the from/until markers,
+    we expect the following logic:
+
+    Marker    Comparison   A->B Included
+    --------- ------------ ---------------
+    From A    fork >= A    True
+    Until A   fork <= A    False
+    From B    fork >= B    True
+    Until B   fork <= B    True
+    """
+    assert BerlinToLondonAt5 >= Berlin
+    assert not BerlinToLondonAt5 <= Berlin
```

Comment:
if berlin to london >= Berlin and   berlintolondon  not <= Berlin then berlintolondon != Berlin ? its tricky because the answer depends on what blocknumber it is. I would rather make transition forks uncomparable to standard forks.

Reply:
Yes exactly, I agree that the comparison doesn't quite make sense between normal forks and transition forks.

These comparisons were used in my previous implementation of this PR, where I had a combination of markers like `valid_from("Cancun")` and `fork_transition_test`, so we needed to compare `Cancun` against all transition forks to see if the test needed to be filled for a given transition fork, but this is not the case anymore.

I can give it a try to modify the behavior to disallow such comparisons.

Reply:
> I can give it a try to modify the behavior to disallow such comparisons.

Yes that is correct, they are treated as different kinds of tests.